src/components/Calendar/doubleCalendar/index.js

```diff
@@ -149,107 +164,119 @@ export default function DoubleCalendar(props) {
 
     return (
         <StyledSection id={id} className={className} style={style} data-calendar-type="double">
-            <StyledCalendarWrapper>
-                <StyledControlsContainer>
-                    <StyledArrowButton
-                        onClick={prevMonthClick}
-                        disabled={disablePreviousMonth}
-                        size="medium"
-                        icon={<LeftIcon />}
-                        assistiveText="Previous Month"
+            <StyledControlsContainer>
+                <StyledArrowButton
+                    ref={headerElementsRefs[0]}
+                    onClick={prevMonthClick}
+                    disabled={disablePreviousMonth}
+                    size="medium"
+                    icon={<LeftIcon />}
+                    assistiveText="Previous Month"
+                    onKeyDown={handleHeaderElementKeyDown}
```

Comment:
I think you should move the onKeyDown call to the parent node just like you did in CalendarHeader, so you can remove the onKeyDown from the YearSelect